charts/observability-pipelines-worker/values.yaml

```diff
@@ -39,6 +39,25 @@ datadog:
     # if you change this port, you'll need to update the livenessProbe and readinessProbe
     address: "0.0.0.0:8686"
 
+bootstrapBackend:
+  filename: bootstrap.yaml
+
+  # exactly one of these should be true
+  secretFile:
+    enabled: false
+
+    # where to mount secrets.json inside the container
+    mountPath: /etc/observability-pipelines-secrets/secrets.json
+
+    # Build secrets.json from entries
+    entries: {}
+
+  custom:
```

Comment:
Links to Docs or comments/examples would be helpful here -- what is a custom secrets backend? What values can I give for `backend_type`? I suspect this is coming and will be in our docs, but without any context I am left wondering when and why I use one vs the other.

Reply:
Good point. The idea was that this is for any value that isn't `file`. Would writing comment in-line here be the place where users would look for documentation too?